workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/utils/api.ts

```diff
@@ -971,11 +981,68 @@ export async function updateUserPassword(password: string, id: number) {
 	const response = await fetch(
 		`/o/headless-admin-user/v1.0/user-accounts/${id}`,
 		{
-			body: JSON.stringify({password}),
+			body: JSON.stringify({ password }),
 			headers,
-			method: 'PATCH',
-		}
+			method: "PATCH",
+		},
 	);
 
 	return response.json();
 }
+
+export async function sendRoleAccountUser(
+	accountId: number,
+	roleId: number,
+	userId: number,
+) {
+	await fetch(
+		`/o/headless-admin-user/v1.0/accounts/${accountId}/account-roles/${roleId}/user-accounts/${userId}`,
+		{
+			headers: {
+				"Content-Type": "application/json",
+				"accept": "application/json",
+				"x-csrf-token": Liferay.authToken,
+			},
+			method: "POST",
+		},
+	);
+}
+
+export async function updateUserImage(userId: number, formData: FormData) {
+	await fetch(
+		`${baseURL}/o/headless-admin-user/v1.0/user-accounts/${userId}/image`,
+		{
+			method: "POST",
+			headers: {
+				"X-CSRF-Token": headers["X-CSRF-Token"],
+			},
+			body: formData,
+		},
+	);
+}
+
+export async function updateMyUserAccount(
+	userId: number,
+	formData: UserForm,
+): Promise<UserAccount> {
+	const response = await fetch(
+		`${baseURL}/o/headless-admin-user/v1.0/user-accounts/${userId}`,
+		{
+			headers,
+			method: "PATCH",
+			body: JSON.stringify(formData),
+		},
+	);
+
+	const accountBriefs = formData.accountBriefs || [];
+
+	for (const account of accountBriefs) {
+		account.roleBriefs.forEach(async (roleBrief: any) => {
```

Comment:
Can you to chagnge this type ?

```suggestion
		account.roleBriefs.forEach(async (roleBrief: RoleBrief) => {
```